tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,31 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a process running on an individual node.
```

Comment:
Maybe @igomez06 can speak more to this...does it matter if the comment is one line like this or do we want to break this up to another line like we do with other functions in the framework?

Either way, I like that it's more descriptive than before!

Reply:
I'll just split into multiple lines to keep it consistent with other comments